symlink.js

```diff
@@ -10,14 +10,14 @@ module.exports = Symlink
 function Symlink (dest) {
   if (this == null) return new Symlink(dest)
   if (dest == null || dest === '') throw new Error('Symlinks must have a destination')
-  Entry.call(this, 'symlink', dest)
+  Entry.call(this, 'symlink', path.join(dest))
 }
 inherits(Symlink, Entry)
 
 Symlink.prototype.create = function (where) {
   var filepath = path.resolve(where, this.path)
   var dest = this.contents
-  if (dest[0] === '/') {
+  if (dest[0] === path.sep) {
```

Comment:
This could also be something like `C:\` which we'd want to treat as a match here.